fleetspeak/test-package.sh

```diff
@@ -16,10 +16,10 @@
 set -e
 
 /bin/echo 'Installing newly built server package.'
-dpkg -i $1
+apt install $1
 
 /bin/echo 'Running fleetspeak-config'
-/usr/bin/fleetspeak-config --config_file=/usr/share/doc/fleetspeak-server/example.config
+/usr/bin/fleetspeak-config --config=/etc/fleetspeak-server/configurator.config
 
 /bin/echo 'ls -l /etc/fleetspeak-server'
 ls -l /etc/fleetspeak-server
```

Comment:
Maybe add a check that the fleetspeak-server process is running?

Reply:
Apparently Trusty and systemd are not friends. We should update our build process to use Xenial, Trusty is too old anyway. I've commented out the check and will uncomment it as soon as Xenial is used.